acceptance/install/scenario3_test.go

```diff
@@ -82,7 +82,7 @@ var _ = Describe("<Scenario3> RKE, Private CA, Configuration, on External Regist
 				"--create-namespace", "metallb", "metallb/metallb")
 			Expect(err).NotTo(HaveOccurred(), out)
 
-			out, err := proc.RunW("kubectl", "apply", "-f", testenv.TestAssetPath("resources.yaml"))
+			out, err = proc.RunW("kubectl", "apply", "-f", testenv.TestAssetPath("resources.yaml"))
```

Comment:
@andreas-kupries could you please give me your opinion here?:
This implementation made yesterday produces an error not finding the `metallb-system` namespace from file `resources.yaml` ([link](https://github.com/epinio/epinio/runs/7324331080?check_suite_focus=true#step:12:140)):

```
Error from server (NotFound): error when creating "../../assets/tests/resources.yaml": namespaces "metallb-system" not found
```
I am not sure if it is simply that needs a bit more of time to create it or that the namespace within the `resource.yaml` file should be changed from `metallb-system` to what is in the test on previous step: `metallb`:
![image](https://user-images.githubusercontent.com/37271841/178924118-88ba2d3a-8fa4-45e0-aa81-64677464ab5d.png)
Or maybe I am missing something else.
Any thoughts?